KalturaWowzaServer/src/main/java/com/kaltura/media/server/wowza/listeners/LiveStreamEntry.java

```diff
@@ -557,6 +558,32 @@ private String getEntryIdFromStreamName(String streamName) {
 		return matcher.group(1);
 	}
 
+		private String getRtmpUrlParameters(String rtmpUrl) {
+
+			final String OldPattern= "rtmp:\\/\\/.+\\.kaltura\\.com:1935\\/kLive\\/\\?(p=[0-9]+)&(e=[01]_[\\d\\w]{8})&(i=0)&(t=[\\d\\w]+)";
```

Comment:
don't use kaltura.com:1935 in the pattern, since this will break everybody that uses different hostname (like QA or developers).
search just from /kLive/....